File: submissions/sm_012_gaurav/week_03/day_3/session_1/print_odds.js
//Function to print odd numbers from 0 to given limit.
function printOddNumbers(limit) {
    for (var i=0;i<=limit;i++) {
        if(i%2 != 0) {
            console.log(i);
        }
    }
}

//Calling printOddNumbers function.
printOddNumbers(14);

File: submissions/sm_009_chinmay/week_13/day_3/session_1/print_odds.py
# Print the odd numbers from 0 to given limit

print('Enter limit')
limit = input()

for x in range(limit+1):
    if x%2!=0:
        print(x)

# Print the odd numbers from given limit to 0
# while limit>=0:
#     if limit%2==1:
#         print(limit)
#     limit-=1